products/early_access_features/frontend/earlyAccessFeatureLogic.ts

```diff
@@ -81,6 +84,44 @@ export const earlyAccessFeatureLogic = kea<earlyAccessFeatureLogicType>([
                 return result
             },
         },
+
+        personsCount: [
+            null as [number, number] | null,
+            {
+                loadEarlyAccessFeatureSuccess: async (_, breakpoint) => {
+                    // Should exist because it was a success, but Typescript doesn't know that
+                    if (!values.earlyAccessFeature || !('feature_flag' in values.earlyAccessFeature)) {
+                        return null
+                    }
+
+                    if (values.personsCount === null) {
+                        await breakpoint(100)
+                    } else {
+                        await breakpoint(1000)
+                    }
+
+                    const results = await Promise.all(
+                        ['true', 'false'].map((value) =>
+                            performQuery<ActorsQuery>({
+                                kind: NodeKind.ActorsQuery,
+                                properties: [
+                                    {
+                                        key: values.featureEnrollmentKey,
+                                        type: PropertyFilterType.Person,
+                                        operator: PropertyOperator.Exact,
+                                        value: [value],
+                                    },
+                                ],
+                                select: ['count()'],
+                            })
+                        )
+                    )
```

Comment:
Maybe I could do this with a single query and a breakdown? I don't know enough about our queries to know that yet, happy to change if that's possible